episodes/11-exercises.md

```diff
@@ -123,7 +121,7 @@ LOs should be tailored to the audience, and,
 if this is well done, you may stop worrying about the audience.
 Create LOs for the specific audience and create assessments for specific LOs."*
 
-\- Dr. Allegra Via, Carpentries Instructor Trainer
+\- Dr. Allegra Via, The Carpentries Instructor Trainer
```

Comment:
"Carpentries Instructor Trainer" (without "The") is fine according to [the style guide](https://docs.carpentries.org/topic_folders/communications/resources/style-guide.html#carpentries), and helps avoid ambiguity about whether Allegra is our only Instructor Trainer or not! 😅

Reply:
👍